Apps/Admin/Client/Utils/AgentUtility.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace HealthGateway.Admin.Client.Utils
+{
+    using HealthGateway.Admin.Common.Constants;
+
+    /// <summary>
+    /// Utilities for interacting with admin agents.
+    /// </summary>
+    public static class AgentUtility
+    {
+        /// <summary>
+        /// Returns the formatted representation of an agent's identity provider.
+        /// </summary>
+        /// <param name="identityProvider">The agent's identity provider.</param>
+        /// <returns>A string containing the formatted representation of a agent's identity provider.</returns>
+        public static string FormatKeycloakIdentityProvider(KeycloakIdentityProvider identityProvider)
```

Comment:
Is there a unit test for this?

Reply:
No, but we don't have unit tests on the Admin.Client project yet.